docs/rules.md

```diff
@@ -340,3 +341,4 @@ TODO: A lot of these are generated so this will need expanded with each unique c
 | Done? | MsgType | Rule name | Addon type | Description | File Type | Source ref | Old Code | New Code |
 | ----- | ------- | --------- | ---------- | ----------- | --------- | ---------- | -------- |
 | :white_check_mark: | warning | N/A |  | manifest_version in manifest.json is not valid. | | null | MANIFEST_VERSION_INVALID |
+| :white_check_mark: | warning | N/A |  | name in manifest.json is not valid. | | null | MANIFEST_NAME_INVALID |
```

Comment:
Why is this a warning and not an error? Seems like something is either valid or not, so it should be an error if it isn't valid.